src/file_saver.rs

```diff
@@ -0,0 +1,89 @@
+// Copyright 2013-2018, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+use glib::object::IsA;
+use glib::translate::*;
+use gtk_source_sys;
+use std::boxed::Box;
+use std::ptr;
+use FileSaver;
+
+impl FileSaver {
+    pub fn save_async<
```

Comment:
This should be in a `FileSaverExtManual` trait so that it can also work on all subclasses. It seems like it's not a final class, or is it?

Reply:
(See comments under `file_loader.rs` above)